src/models/speed.h

```diff
@@ -0,0 +1,93 @@
+//
+// Created by Julien on 19/06/2023.
+//
+
+#ifndef PACMAN_SPEED_H
+#define PACMAN_SPEED_H
+
+#include "game.h"
+#include <limits.h>
+
+enum PacmanSpeed
+{
+    PACMAN_NORM_SPEED,
+    PACMAN_NORM_DOTS_SPEED,
+    PACMAN_FRIGHT_SPEED,
+    PACMAN_FRIGHT_DOTS_SPEED
+};
+
+enum GhostSpeed
+{
+    GHOST_NORM_SPEED,
+    GHOST_FRIGHT_SPEED,
+    TUNNEL_SPEED
+};
+
+typedef struct
+{
+    int minRound;
+    int maxRound;
+    float normSpeed;
+    float normDotsSpeed;
+    float frightSpeed;
+    float frightDotsSpeed;
+} PacmanSpeedLevel;
+
+typedef struct
+{
+    int minRound;
+    int maxRound;
```

Comment:
I think that maxRounds are useless. We can get speed like this :
``
(level) => {
    i = 0
    while (speeds[i+1].minLevel <= level) {
       i++;
    }
   return speeds[i];
}
```

Reply:
ok, but it's false to do `speeds[i+1]`